Documentation/faq.md

```diff
@@ -10,6 +16,24 @@
 
 ### Deployment
 
+#### System requirements
+
+Since etcd writes data to disk, SSD is highly recommended. Default storage size limit is 2GB, configurable up to 8GB. 2GB is a reasonable limit for most use cases. And allocate as much heap size to avoid disk swap. If the etcd storage quota is 2GB, the machine should have at least 2GB of RAM. At CoreOS, etcd cluster is usually deployed on dedicated CoreOS Container Linux machines, with dual-core processors, 2GB of RAM, and 80GB of SSD *at the very least*. **It highly varies on workloads; please test before production deployment**.
```

Comment:
`Since etcd writes data to disk, SSD is highly recommended. To prevent performance degradation or unintentionally overloading the key-value store, etcd enforces a 2GB default storage size quota, configurable up to 8GB. To avoid swapping or running out of memory, the machine should have at least as much RAM to cover the quota. At CoreOS, an etcd cluster is usually deployed on dedicated CoreOS Container Linux machines with dual-core processors, 2GB of RAM, and 80GB of SSD *at the very least*. **Note that performance is intrinsically workload dependent; please test before production deployment**.`

have this link to the recommended hardware document somewhere?